pkg/config/controller_config.go

```diff
@@ -70,6 +71,16 @@ func ConfigInit() {
 		glog.V(2).Infoln("CLUSTER_LOCAL_GATEWAY", DefaultServiceNetwork)
 	}
 
+	tgNameLengthMode := os.Getenv("TARGET_GROUP_NAME_LEN_MODE")
```

Comment:
Are we going to enable `long` mode by default in the future?

Reply:
Right now, our e2e tests are using shortname.  We have to wait and see